packages/menu/README.md

```diff
@@ -104,63 +104,67 @@ import { Menu, MenuSeparator, MenuItem } from '@leafygreen-ui/menu';
-
-_Any other properties will be spread on the Menu `div` container_
+| Prop               | Type                                           | Description                                                                                                                                                                                                                                                             | Default      |
+| ------------------ | ---------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------ |
+| `open`             | `boolean`                                      | Determines whether or not the `<Menu />` will appear as open or closed                                                                                                                                                                                                  | `false`      |
+| `setOpen`          | `function`                                     | When controlling the component, use `setOpen` to keep track of the `<Menu />` component's state so that clicks on the document's backdrop as well as a user pressing the Escape Key will close the Menu and update the consuming application's local state accordingly. |              |
+| `shouldClose`      | `function`                                     | Determines if the `Menu` should close when the backdrop or Escape keys are clicked. Defaults to true.                                                                                                                                                                   | `() => true` |
+| `align`            | `'top'` \| `'bottom'` \| `'left'` \| `'right'` | Determines the alignment of the `<Menu />` component relative to a reference element, or the element's nearest parent                                                                                                                                                   | `'bottom'`   |
+| `justify`          | `'start'` \| `'middle'` \| `'end'`             | Determines the justification of the `Menu` component (based on the alignment) relative to a reference element or the element's nearest parent                                                                                                                           | `'end'`      |
+| `refEl`            | `node`                                         | Pass a reference to an element that the `Menu` component should be positioned against                                                                                                                                                                                   |              |
```

Comment:
We've been using `node` to mean `React.ReactNode`. in that sense, this is incorrect. this should be an `HTMLElement`